client/public/styles/editTemplate.scss

```diff
@@ -0,0 +1,126 @@
+body {
+  background-color: #eee;
+  font-family: Arvo, serif;
+  font-weight: bold;
+  color: #003366;
+}
+.edit-template-div {
+  background-color: white;
+  text-align: center;
+  width: 836px;
```

Comment:
We want to use % for width values, generally.

Reply:
Issue when the width is a % for .edit-template-div:
![Screen Shot 2020-05-28 at 8 21 31 PM](https://user-images.githubusercontent.com/55335661/83207322-ef0f3c80-a120-11ea-9331-41749404e646.png)